src/main/java/core/basesyntax/Main.java

```
@@ -0,0 +1,195 @@
+                break;
+            case "info":
+                info(new File(command[1], command[2]));
+                break;
+            case "help":
+                help(command);
+                break;
+            case "exit":
+                System.exit(0);
+                break;
+            default:
+                noCommand(input);
+                break;
+        }
+    }
+
+    public static void create(File file) {
+        try {
+            boolean created = file.createNewFile();
+            if (created) {
+                System.out.println("Файл создан");
+            } else {
+                System.out.println("Файл уже существует. Хотите перезаписать? Y - Да, N - Нет");
+                switch (new Scanner(System.in).nextLine().toUpperCase()) {
+                    case "Y":
+                        file.delete();
+                        System.out.println(file.createNewFile() ? "Файл перезаписан" :
+                                "Системе не удалось сделать перезапись");
+                        break;
+                    case "N":
+                        System.out.println("Вы отменили перезапись файла");
+                        break;
+                    default:
+                        System.out.println("Неизвестная команда!");
+                        break;
+                }
+            }
+        } catch (IOException ex) {
+            System.out.println(ex.getMessage());
+        }
+    }
+
+    public static String read(File file) {
+        String read = "";
+        try (FileReader fileReader = new FileReader(file);
+                Scanner readScanner = new Scanner(fileReader)) {
+            while (readScanner.hasNextLine()) {
+                read += readScanner.nextLine() + "\n";
+            }
+        } catch (FileNotFoundException e) {
+            System.out.println("Такого файла по заданному пути не существует");
+        } catch (IOException e) {
+            e.printStackTrace();
```

do not ignore exceptions